src/components/custom-select.scss

```diff
@@ -61,7 +74,7 @@
       height: 24px;
       width: 24px;
       margin: 0 5px 0 5px;
-      fill: $cc-teal;
+      fill: var(--theme-primary-color);
```

Comment:
Same as above.  Be cautious when using the theme colors against static backgrounds.  This won't show up with the Interactions theme.  I'm assuming at some point we will get actual theme color specs so we don't need to stress about this now, but something that we should remember to return to when we work on theme colors again.